subsystems/drivebase/SwerveDrive.java

```diff
@@ -288,28 +330,36 @@ public void initCoastMode() {
 
     public void toggleIdleMode() {
         for (SwerveModule m : mModules) {
-            if (m.getDriveMotor().getIdleMode() == IdleMode.kBrake)
-                    m.getDriveMotor().setIdleMode(IdleMode.kCoast);
+            if (((CANSparkMax) m.getDriveMotor()).getIdleMode() != IdleMode.kBrake)
```

Comment:
I don't know why we're touching this in this change but it'll break with the kraken implementation